[libevdev] Unable to install

It looks like `doxygen` and `python27` dependencies are missing.
```
checking for doxygen... no
configure: WARNING: doxygen not found - required for documentation
checking whether to build with gcov... no
checking for a Python interpreter with version >= 2.6... none
configure: error: no suitable Python interpreter found
libevdev failed to install: `./configure --prefix=/usr/local --libdir=/usr/local/lib` exited with 1
```

--- a/packages/libevdev.rb
+++ b/packages/libevdev.rb
@@ -7,6 +7,9 @@ class Libevdev < Package
   source_url 'https://www.freedesktop.org/software/libevdev/libevdev-1.5.9.tar.xz'
   source_sha256 'e1663751443bed9d3e76a4fe2caf6fa866a79705d91cacad815c04e706198a75'
 
+  depends_on 'doxygen'
+  depends_on 'python27'
+  
   def self.build
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
     system "make"

--- a/packages/libevdev.rb
+++ b/packages/libevdev.rb
@@ -3,7 +3,7 @@ require 'package'
 class Libevdev < Package
   description 'libevdev is a wrapper library for evdev devices.'
   homepage 'https://www.freedesktop.org/wiki/Software/libevdev'
-  version '1.5.9'
+  version '1.5.9-0'
   source_url 'https://www.freedesktop.org/software/libevdev/libevdev-1.5.9.tar.xz'
   source_sha256 'e1663751443bed9d3e76a4fe2caf6fa866a79705d91cacad815c04e706198a75'
 

